config/src/sumeragi.rs

```diff
@@ -24,13 +24,12 @@ view! {
     /// `Sumeragi` configuration.
     /// [`struct@Configuration`] provides an ability to define parameters such as `BLOCK_TIME_MS`
     /// and a list of `TRUSTED_PEERS`.
-    #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, Documented, Proxy, LoadFromEnv)]
-    #[serde(default)]
+    #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, Proxy, Documented, LoadFromEnv)]
     #[serde(rename_all = "UPPERCASE")]
     #[config(env_prefix = "SUMERAGI_")]
     pub struct Configuration {
         /// The key pair consisting of a private and a public key.
-        #[serde(skip)]
+        //TODO: Why skip was here?
```

Comment:
Maybe because keys was substituted for sumeragi in `finalization` stage?

Reply:
Keys must be substituted later. So deserializing them here makes no sense.

Reply:
Still it seems that line is unneeded on the `Configuration` now (and trying to put it there complains about a lack of `Default` impl), but might need to be put on proxy specifically. Also as a side question for sake of clarification: should we ditch the `Deserialize` derive on `Configuration` altogether? It feels like that responsibility is completely superceded by the `Proxy`, but I might be missing some other points, so it needs checking.

Reply:
WIth `#[serde(skip)]` on proxy field you could just don't check `sumeragi_proxy.key_pair.is_some()` because you know that it will be `None`.